### ISSUE TITLE
Breakpoints are being ignored

### ISSUE THREAD

**Issue by Yaskier** [sag1v/react-elastic-carousel#69]
**Describe the bug**
The breakpoints are ignored, always showing 1 item.

**To Reproduce**
```
export const NewsCarousel: FC<{}> = () => {
  const news = [
    {
      title: 'test1',
    },
    {
      title: 'test2',
    },
    {
      title: 'test3',
    },
    {
      title: 'test4',
    },
  ];

  const breakpoints = [
    { width: 1, itemsToShow: 1, itemsToScroll: 1 },
    { width: 550, itemsToShow: 2, itemsToScroll: 1 },
    { width: 850, itemsToShow: 3, itemsToScroll: 1 },
    { width: 1150, itemsToShow: 4, itemsToScroll: 1 },
    { width: 1450, itemsToShow: 4, itemsToScroll: 1 },
    { width: 1750, itemsToShow: 4, itemsToScroll: 1 },
  ];

  return (
    <div>
      <Carousel
        //itemsToShow={4}
        breakpoints={breakpoints}
        showArrows={false}
        itemPadding={[0, 8]}
        enableMouseSwipe={false}
      >
        {news.map(aNew => (
          <div key={aNew.title}>
            <div>{aNew.title}</div>
          </div>
        ))}
      </Carousel>
    </div>
  );
};
```

**Expected behavior**
I expected it to show 4 items and decrease the quantity when the screen is smaller

**Screenshots**
![image](https://user-images.githubusercontent.com/1658525/90160798-8ab06080-dd92-11ea-89ec-cefc9cfe6703.png)

**Desktop (please complete the following information):**
- Ubuntu
-Chrome

**Comment by sag1v** [comment]
Hi @Yaskier, you have a typo in the `breakPoints` prop (note the upper case `P` for Points).

Instead of `breakpoints` you should pass `breakPoints` as prop.

**Comment by Yaskier** [comment]
That's not a typo, maybe is a bug in your typings, I already saw that difference in your documentation, but the typescript typings definitions force me to write it that way. 

I can provide proof if you need it, please open the issue again.

**Comment by sag1v** [comment]
@Yaskier Sorry i guess misunderstood. Anyway thanks, i fixed the typo in the typing file and published the fix under version `0.7.2`. Let me know how it goes 🙂

**Comment by sag1v** [comment]
@Yaskier Is the latest version working well for you?
